src/shared/messages.ts

```diff
@@ -131,7 +131,11 @@ export enum BackgroundToContentAction {
 
 export interface MonetizationEventPayload {
   requestId: string
-  details: any
+  details: {
```

Comment:
If transforming the data in background
```suggestion
  detail: {
```